moon/autorun/server/cfc_leftover_chips.moon

```diff
@@ -29,13 +30,15 @@ onCreate = (ent) ->
 
         ent\CallOnRemove "CFC_Leftovers_Untrack", ->
             rawset leftovers, ent, nil
-hook.Add "OnEntityCreated", "CFC_Leftovers_Track", onCreate
 
 cleanup = ->
     for leftover in pairs leftovers
         if IsValid leftover
-            property = rawget checkProperty, leftover\GetClass!
-            continue if IsValid leftover[property]
+            getter = rawget getOwner, leftover\GetClass!
+            continue unless getter
+            continue if IsValid getter leftover
 
         SafeRemoveEntityDelayed leftover, 0
```

Comment:
```suggestion
    timer.Simple 1, ->
        for leftover in pairs leftovers
            if IsValid leftover
                getter = rawget getOwner, leftover\GetClass!
                continue unless getter
                continue if IsValid getter leftover

            SafeRemoveEntityDelayed leftover, 0
```

Reply:
The timer's actually on the hook.Add at the bottom, but I realized it needs to be 1s not 0